ocs_ci/templates/workloads/fio/workload_io_fillup.yaml

```diff
@@ -0,0 +1,10 @@
+name: fio-fillup
+filename: fio-fill-up
+rw: write
+bs: 1m
+direct: 1
+numjobs: 1
+time_based: 0
+runtime: 36000
```

Comment:
should we have such a time run time as default? If in case there are any failures, the test might take such a long time to report failure.

Reply:
The IO will run until all data was written **or** up to the time is finished, this runtime if high so all data will be written for sure.
If the is error while writing and the test failed, it will not continue this long time.